orttraining/orttraining/training_api/module.cc

```diff
@@ -194,6 +195,17 @@ size_t Module::GetEvalModeOutputCount() const noexcept {
   return eval_output_names_.size();
 }
 
+size_t Module::GetParametersSize(const bool trainable_only) const noexcept {
+  size_t parameters_size = 0;
+  for (auto& it : named_parameters_) {
+    if (trainable_only && !it.second->RequiresGrad()) {
+      continue;
+    }
+    parameters_size += it.second->Data().Get<Tensor>().Shape().Size();
```

Comment:
I am guessing it will almost certainly never happen, but can we check for overflow?